i18n/en.json

```diff
@@ -3866,6 +3866,10 @@
     "id": "ent.compliance.csv.metadata.json.marshalling.appError",
     "translation": "Unable to convert metadata to json."
   },
+  {
+    "id": "ent.compliance.csv.metadata.json.zipfile.appError",
+    "translation": "Unable to create the zip file"
```

Comment:
I thought CI should fail if a PR missed a translation string. Any idea why `master` is not broken?

Reply:
This is from the EE PR for staticcheck. I added a new translation there.